Sources/Sentry/SentrySDK.m

```diff
@@ -160,7 +160,7 @@ + (void)startWithOptions:(SentryOptions *)options
     // We accept the tradeoff that the SDK might not be fully initialized directly after
     // initializing it on a background thread because scheduling the init synchronously on the main
     // thread could lead to deadlocks.
-    SENTRY_LOG_DEBUG(@"Starting SDK...");
+    SENTRY_LOG_DEBUG(@"Starting SDK with options: %@", options.debugDescription);
```

Comment:
`m`: I thought it would be enough to just pass the `options` here, and the NSString.format would use the `debugDescription`, but it doesn't. The `debugDescription` works only when the debugger is attached. So I think it's better to revert to your old way of using `#if defined(DEGUG)` in the description, because now `debugDescription` will also run in release, and I don't think that's needed. Or we simply change `debugDesription` to `description` that's also fine. Up to you.

Reply:
I'm going to leave it as `debugDescription` and will surround it with the ifdef